core/src/components/modal/test/a11y/modal.spec.ts

```diff
@@ -1,23 +0,0 @@
-import { newSpecPage } from '@stencil/core/testing';
-
-import { Modal } from '../../modal';
-
-describe('modal: a11y', () => {
```

Comment:
I wanted to add tests for all of the inherited roles, and I also wanted to test the htmlAttributes behavior that I added in this PR. As a result, I decided to remove this test in favor of more generic attribute tests.